src/util/networkUtils.js

```diff
@@ -1,7 +1,28 @@
 const fs = require('fs')
 const path = require('path')
 
-async function getNetworkInfo (buildPath) {
+const NETWORK_IDS = {
```

Comment:
Can we not keep these on the `const networks` object that gets exported by the truffleConfig and access it from there somehow?